main.js

```diff
@@ -13,32 +13,37 @@ var render_lists = function (lists) {
     .join("");
   list.innerHTML = ItemList;
 };
-render_lists(tasks);
+renderLists(tasks);
 //task filter section
 const render = () => {
   searchBar.addEventListener("keyup", (task) => {
     const searchString = task.target.value.toLowerCase();
     const filteredTasks = tasks.filter((task) =>
       task.name.toLowerCase().includes(searchString)
     );
-    render_lists(filteredTasks);
+    renderLists(filteredTasks);
   });
 };
 render();
 //task sorting section
-document.querySelector("#ascending").addEventListener("click", () =>
+document.querySelector("#ascending").addEventListener("click", () => {
   tasks.sort(function (a, b) {
     if (a.name < b.name) {
       return -1;
     }
-    render_lists(tasks);
-  })
-);
-document.querySelector("#descending").addEventListener("click", () =>
+  });
+  renderLists(tasks);
+});
+document.querySelector("#descending").addEventListener("click", () => {
   tasks.sort(function (a, b) {
     if (a.name > b.name) {
       return -1;
     }
-    render_lists(tasks);
-  })
-);
+  });
+  renderLists(tasks);
+});
+function loaded() {
```

Comment:
Try another way of opening the first task, do it pragmatically i.e using a boolean property. Also the current active task is not highlighted properly